rpc/gen_proto.sh

```diff
@@ -23,50 +23,50 @@ protoc -I common common/security.proto --go_out=plugins=grpc:$GOPATH/src
 
 Module="admin"
 echo "Generating $Agent/$Module protoc"
-protoc -I $Agent/$Module/credentialmonitor -I ./common -I $GOPATH/src/github.com/protocolbuffers/protobuf/src $Agent/$Module/credentialmonitor/credentialmonitor.proto  --go_out=plugins=grpc:$GOPATH/src
```

Comment:
Through experimentation I discovered that `protoc` will check the `PATH` environment variable for any dependencies it needs. Taking advantage of this means we can remove the dependency on having these files live in a specific directory.